cmd/neofs-cli/internal/common/exit.go

```diff
@@ -9,10 +9,16 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// ExitOnErr prints error and exits with a code that matches
-// one of the common errors from sdk library. If no errors
-// found, exits with 1 code.
-// Does nothing if passed error in nil.
+// ErrAwaitTimeout represents the expiration of a polling interval
+// while awaiting a certain condition.
+var ErrAwaitTimeout = errors.New("await timeout expired")
+
+// ExitOnErr prints error and exits with a code depending on the error type
+//
+//	0 if nil
+//	1 if [sdkstatus.ErrServerInternal] or untyped
+//	2 if [sdkstatus.ErrObjectAccessDenied]
+//	3 if [ErrAwaitTimeout]
```

Comment:
We need some user-facing document with this. Like `docs/cli-return-codes.md` referenced from `README.md`.

Reply:
@roman-khimov can you pls provide an example? also, regular users won't see any codes only error text. is this doc should be user-friendly and contain info/examples of commands outputs and their codes?

Reply:
Exit code is a part of a program API, programs like `neofs-cli` can be (and often are!) used in scripts where exit codes matter. We just need a simple table "code -> meaning". Like "Exit codes" section in [`man wget`](https://www.man7.org/linux/man-pages/man1/wget.1.html), for example.

Reply:
done